virt/virt-4-13-release-notes.adoc

```diff
@@ -90,6 +91,8 @@ The SVVP Certification applies to:
 
 //CNV-25574 Release note: NEW Ability to make a hot-plug disk part of the VM
 
+//CNV-22574 Release note: NEW UI - Ability to make a hot-plug disk part of the VM
```

Comment:
Looks like a duplicate of the placeholder on line 92?

Reply:
I removed one of the instances from the file.

Reply:
It looks like CNV-2**5**574 is the correct Jira ID for this one, not CNV-2**2**574.

Reply:
Fixed it; thanks!